libcudacxx/cmake/LibcudacxxInternalHeaderTesting.cmake

```diff
@@ -45,8 +45,8 @@ list(FILTER internal_headers EXCLUDE REGEX "__ptx/instructions/generated")
 
 function(libcudacxx_create_internal_header_test header_name headertest_src)
   # Create the default target for that file
-  set(internal_headertest_${header_name} verify_${header_name})
```

Comment:
Drive-by clean-up -- this has no effect, the created variable is never used. Same for the other libcu++ header test files.